### PR TITLE
feat(cli): add --secrets flag to task run

### PR DESCRIPTION
**Pull request by Lou1415926**
Allow user to specify a list of secrets they want to inject into the container.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by iamhopaul123 — COMMENTED** [review]
The code LGTM. Do we need to make our unit tests cover this secret usage?

File: internal/pkg/cli/flag.go [hunk 1]
```diff
@@ -213,6 +214,7 @@ Must be of the format '<keyName>:<dataType>'.`
 	taskRoleFlagDescription      = "Optional. The ARN of the role for the task to use."
 	executionRoleFlagDescription = "Optional. The ARN of the role that grants the container agent permission to make AWS API calls."
 	envVarsFlagDescription       = "Optional. Environment variables specified by key=value separated with commas."
+	secretsFlagDescription       = "Optional. Secrets to inject into the container. Specified by key=value separated with commas."
```
Comment (huanjani) [inline internal/pkg/cli/flag.go]: nit: can we change "separated with commas" to "separated by commas" here and for the envVars and resourceTags flag descriptions?

File: site/content/docs/commands/task-run.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ Generally, the steps involved in task run are:
   --memory int                     Optional. The amount of memory to reserve in MiB for each task. (default 512)
   --resource-tags stringToString   Optional. Labels with a key and value separated with commas.
                                    Allows you to categorize resources. (default [])
+  --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated with commas. (default [])
```
Comment (huanjani) [inline site/content/docs/commands/task-run.md]: nit: same here, "with" -> "by"? And for `--resource-tags` and `--env-vars`, too?

**Review by huanjani — APPROVED** [review]
Looks great! Thanks for making those extra fixes, too!

**Review by efekarakus — APPROVED** [review]
Looks great! Just to make sure, did we test this both with SSM secrets and secret manager secrets?

**Pull request merged**